### PR TITLE
ENHANCEMENT - Added some TL;DR and some info if using a linked spree-ember

### PR DESCRIPTION
**Pull request by triptec**
I can see how this might not be the place but it might be helpful, you decide ;)

### CODE REVIEW

File: docs/guides/3-getting-started.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ First - you'll need the Ember CLI.  Head over to
 [Getting Started with Ember CLI](http://www.ember-cli.com/#getting-started) for 
 instructions.
 
+**TL;DR**
+
+```bash
+npm install -g ember-cli
+```
+
```
Comment (hhff) [inline docs/guides/3-getting-started.md]: First - you'll need the Ember CLI.  You can install it like so:

```bash
npm install -g ember-cli
```

Head over to [Ember CLI](http://www.ember-cli.com/) for more details.

Comment (hhff) [inline docs/guides/3-getting-started.md]: this is such an easy install we might aswell include it, like so^^

File: docs/guides/2-configuring-spree.md [hunk 2]
```diff
@@ -26,6 +35,21 @@ namespace to Spree's default API routes, that respond as per the
...
+echo "\ngem 'spree_ams', :github => 'hhff/spree_ams', :branch => '3-0-alpha'" >> Gemfile
+bundle
+rails g spree:api:ams:install
+```
+
+In ```config/environments/development.rb``` add:
+```
+config.action_controller.asset_host = 'http://localhost:3000'
+```
+
```
Comment (hhff) [inline docs/guides/2-configuring-spree.md]: I'd like to remove these parts, as they're locked to specific versions (the rails version often matters here too), and may go out-of-date.  I think it's easy enough for a developer to consult the specific page on github, than for us to constantly keep in up-2-date

File: docs/guides/3-getting-started.md [hunk 3]
```diff
@@ -40,6 +46,12 @@ ember install spree-ember-storefront
 This command will copy all of the storefront route & component templates to your 
 application, for easy customization.
 
+If you have linked the spree-ember(http://github.com/hhff/spree-ember) repo with ```bootstrap.rb``` first run this command. 
+```bash
+npm link spree-ember-storefront
+```
+Then add ```"spree-ember-storefront": "*"``` to your package.json
+
```
Comment (hhff) [inline docs/guides/3-getting-started.md]: we can remove this in favor of adding something similar to CONTRIBUTING.md

**Comment by triptec** [comment]
Not much left ;)

**Comment by hhff** [comment]
That's OK!  The ember cli page is kinda intimidating - this might help onboard some folks :)

Can you plz rebase on master?

**Comment by triptec** [comment]
Force pushed by misstake, closing and I'll add that about ember-cli

**Pull request closed without merging**